GSoC25/entity-linking-master/batch_preprocessing/batch_dbpedia_uri.py

```
@@ -0,0 +1,58 @@
+          ?uri rdfs:label ?canonical_name .
+          FILTER (lang(?canonical_name) = 'en')
+        }}
+        '''
+        print(f"[DEBUG] SPARQL Query for batch {i+1}/{n_chunks}:\n", query)
+        sparql.setQuery(query)
+        sparql.setReturnFormat(JSON)
+        try:
+            batch_results = sparql.query().convert()
+            print(f"[DEBUG] Raw SPARQL results for batch {i+1}:\n", batch_results)
+            uri_map = {r["canonical_name"]["value"]: r["uri"]["value"] for r in batch_results["results"]["bindings"]}
+        except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)